theme/theme.inc

```diff
@@ -234,11 +234,19 @@ function islandora_solr_metadata_query_fields($object, &$solr_fields) {
               // Will get stripped out later.
               return;
             }
-            $solr_query = format_string('!field:"!value"', array(
+            $solr_query = format_string('!field:!value', array(
               '!field' => $solr_field,
-              '!value' => urlencode(islandora_solr_replace_slashes(islandora_solr_lesser_escape($original_value))),
+              '!value' => islandora_solr_facet_escape($original_value),
+            ));
+            return t('<a href="@url">@val</a>', array(
+              '@url' => url("islandora/search/", array(
+                "query" => array(
+                  "type" => "dismax",
+                  "f[]" => $solr_query,
```

Comment:
Pretty sure an array here would make more sense... Be alterable and the like. Having `f[]` explicitly be a string would probably harder to work with (to programmatically add more `fq`s, say).

This entire return could become something like:
```php
return l($value, 'islandora/search', array(
  'type' => 'dismax',
  'f' => array(
    $solr_query,
  ),
));
```
or something of the like?